quinn/examples/README.md

```diff
@@ -0,0 +1,83 @@
+## HTTPS File Serving Example
+
+The `https` example shows how to request a file on the `server` from the `client` over HTTPS. 
+This is HTTP/0.9, which is a toy protocol that's much simpler than real HTTP, especially HTTP/3.
+
+(Note: when providing the example server with a specific address to listen on,
+make sure you match the correct IPv4 vs IPv6 address in the client command.)
```

Comment:
So @Ralith said in https://github.com/quinn-rs/quinn/pull/908#discussion_r522644091:

> I'm not sure about this wording. We should note that it listens on IPv6 by default, and I don't think there's anything the client can easily do to change the address family, since a text name is needed for certificate verification.

The easy way out for this PR is just to leave this out so we can discuss it separately -- it doesn't really match the goal of merging the example READMEs anyway.

Reply:
Good idea, removed.